app/controllers/jobs_controller.rb

```diff
@@ -11,8 +11,20 @@ def show
   def create
     @job = JobPosting.new(job_params)
 
+    # Collect and cache information required for future moderation actions
+    @job.attributes = {
+      user_ip: request.env['REMOTE_ADDR'],
+      user_agent: request.env['HTTP_USER_AGENT'],
+      referrer: request.env['HTTP_REFERER']
+    }
+    @job.akismet_spam = @job.spam?
+
     if @job.save
-      redirect_to job_path(@job), notice: "Your job listing has been posted."
+      if @job.akismet_spam
+        redirect_to job_path(@job), alert: "Your job listing has been posted, but requires moderator approval."
+      else
+        redirect_to job_path(@job), notice: "Your job listing has been posted."
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.